src/AddIns/Uno.UI.Lottie/LottieVisualSource.Skottie.cs

```diff
@@ -471,5 +471,10 @@ private Size CompositionSize
 				? new Size(size.Width, size.Height)
 				: default;
 	}
+
+	public partial class UnoSKSwapChainPanel : SKSwapChainPanel
+	{
+		protected override Size ArrangeOverride(Size finalSize) => ArrangeFirstChild(finalSize);
```

Comment:
It might be worth to leave a comment so that this is cleaned up in future and get back to `SKSwapChainPanel`.